synapse/storage/databases/main/event_push_actions.py

```diff
@@ -927,18 +930,19 @@ def _handle_new_receipts_for_notifs_txn(self, txn: LoggingTransaction) -> bool:
 
         # We always update `event_push_summary_last_receipt_stream_id` to
         # ensure that we don't rescan the same receipts for remote users.
-        #
-        # This requires repeatable read to be safe, as we need the
-        # `MAX(stream_id)` to not include any new rows that have been committed
-        # since the start of the transaction (since those rows won't have been
-        # returned by the query above). Alternatively we could query the max
-        # stream ID at the start of the transaction and bound everything by
-        # that.
```

Comment:
what was the reason for removing this comment, ooi?

Reply:
We're no longer reading the `MAX(stream_id)` and instead explicitly bound what we pull from the DB

Reply:
ah duh yes. thanks.